### DISCUSSION TITLE
Override the "Enter" hotkey on Table2

### DISCUSSION THREAD

**Discussion by CannonLock** [palantir/blueprint#6660] [Q&A]
Hello, I am using Table2 to power a editing interface and I would like users to be able to edit a cell on hitting enter. The issue is that this also causes the focused cell to move to the row below.

Is there a way that I can intercept this or overwrite it with another command? I have the focusedCell controlled but I have no way of knowing the event that triggered the change.

**Comment by sandstone991** [comment]
Hi @CannonLock,

Did you find a solution for this?

**Reply by CannonLock** [reply]
I did not, had to handle things another way.
